### ISSUE TITLE
Boundrybox and prediction on video[FEATURE REQUEST]: Title

### ISSUE THREAD

**Issue by bpabla1234** [googlecreativelab/teachablemachine-community#317]
**Is your feature request related to a problem? Please describe.**
I want to have the prediction not on the print but on the boundary box 
**Describe the solution you'd like**
i want  boundry box with prediction name
**Describe alternatives you've considered**
currently i tried to draw rectangle but it is giving me error
**Additional context**

Please note that we cannot guarantee we will work on your feature request at this time. Your input is still very useful in helping us know what to prioritize. Thank you!

**Comment by sryu1** [comment]
Sorry, could you elaborate?

**Comment by bpabla1234** [comment]
Yes it is detecting the object and printing. I want a rectangle drawn on the object detected with label .

So that while camera 🤳 is looking you find your trained object detected live in screen

**Comment by sryu1** [comment]
I think that'll be hard as when training with teachable machine, it uses the full image as the input instead of an image with labels.

**Comment by bpabla1234** [comment]
I agree with you I tried doing this. We can then just draw rectangle on detected object is this possible or write or print the label on image

**Comment by kylejramstad** [comment]
The teachable machine uses Tensorflow image classification. What you are looking for is object detection and involves a lot of image/video annotations.
